docs/src/components/Palette/Palette.less

```diff
@@ -0,0 +1,21 @@
+@import (reference) '../../../../theme';
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.item {
+  flex: 0 0 168px;
```

Comment:
Not too fussed about the magic number on the docs site, but have you confirmed this number is ok on mobile?

Reply:
Good call. Will double check.